R/roclet.R

```diff
@@ -94,6 +94,12 @@ roclet_find <- function(x) {
   env$namespace <- namespace_roclet
   env$vignette <- vignette_roclet
 
+  # Making it possible to specify external roclets
+  # without the _roclet postfix.
+  if (!grepl(pattern = "_roclet$", x)) {
```

Comment:
I'd prefer to encourage people to use the full names. When #889 is implementation, we can remove the special casing for these existing roclets.

Reply:
All good. Thanks for the input.